web/shared/CreateCharacterForm.tsx

```diff
@@ -543,7 +543,7 @@ export const CreateCharacterForm: Component<{
                     label="Post-conversation history instructions (optional)"
                     helperText={
                       <span>
-                        Prompt to bundle with your character. Leave empty if you aren't sure.
+                        {`Prompt to bundle with your character, used at the bottom of the prompt. You can use the {{optional}} placeholder to include the user's UJB, if you want to supplement it instead of replacing it.`}
```

Comment:
`{{original}}` instead of `{{optional}}` :)

Reply:
i have the stupids